src/WebJobs.Script.WebHost/Diagnostics/FunctionInstanceLogger.cs

```diff
@@ -83,9 +83,7 @@ internal object LogInvocationMetrics(FunctionMetadata metadata)
             {
                 string eventName = _bindingMetricEventNames.GetOrAdd(binding, static (b) =>
                 {
-                    return b.IsTrigger ?
-                        string.Format(MetricEventNames.FunctionBindingTypeFormat, b.Type) :
-                        string.Format(MetricEventNames.FunctionBindingTypeDirectionFormat, b.Type, b.Direction);
```

Comment:
Let's remove the now unused format string

Reply:
Removed unused constant.